Readme.md

```diff
@@ -51,6 +51,35 @@ var log = pinoms({streams: streams})
 log.info('this will be written to /tmp/info.stream.out')
 log.fatal('this will be written to /tmp/fatal.stream.out')
 ```
+And to use with [pino-pretty](https://github.com/pinojs/pino-pretty)
+```js
+const fs = require('fs');
+const Pino = require('pino');
+const PinoPretty = require('pino-pretty');
+const PinoGetPrettyStream = require('pino/lib/tools').getPrettyStream;
```

Comment:
Ouch! We shouldn’t be doing this! Maybe we can move that function to pino-pretty itself?

Reply:
That `getPrettyStream` function is very internal to the workings of the `pino` module. It really shouldn't be in another module or used by anyone else.

Reply:
We should fix this somehow ;). What do you propose @jsumners?

Reply:
@mcollina @jsumners Is there a better way to use pino-pretty with multi-stream, that does not require using `getPrettyStream`?

Reply:
There isn't. However we need to expose https://github.com/pinojs/pino/blob/master/lib/tools.js#L180-L265 somehow. That's what pino-multi-stream will need to support prettifiers natively.

Reply:
Would this be something to consider?

https://github.com/pinojs/pino/compare/master...ovhemert:pretty-stream

Adds a new function to wrap the internal `getPrettyStream`. You can then specify your own prettifier and/or destination. Defaults to `pino-pretty` & `stdout`.

Since `pino` functions are also available in `pino-multi-stream`, you can then write the example/doc like this:

``` js
const fs = require('fs');
const pinoms = require('pino-multi-stream')

const prettyStream = pinoms.createPrettyWriteStream()
const streams = [
    {stream: fs.createWriteStream('my.log') },
    {stream: prettyStream }
]

let logger = pinoms(pinoms.multistream(streams))

logger.info("HELLO %s!", "World")

```

Reply:
I really don't know much about the `prettifierMetaWrapper` implementation. That's @mcollina's handiwork. If the proposed patch passes his review then I'm on board with it.

Reply:
I’m +1 with the API. I’d still love to see the metaWrapper documented, but it can wait (and we can close this issue).